api/templates/client.mustache

```diff
@@ -580,10 +580,13 @@ func (e GenericOpenAPIError) BuildHeader() string {
 
 // Error returns non-empty string if there was an error.
 func (e GenericOpenAPIError) Error() string {
+	if e.error != "" {
+		return e.error
+	}
 	if e.model != nil {
 		return e.model.Error()
 	}
-	return e.error
+	panic("errorless GenericOpenAPIError used as an error!")
```

Comment:
I think this change should fix the panic you were seeing with the 422 response, but I am wondering in what situation would we be calling the `e.Error()` function, but both `e.error` and `e.model.Error()` are errorless? We would expect to always have an error in one or the other, right?